examples/app/uploadservice-s3/src/main/java/com/example/uploadservice/s3/S3UploadTask.kt

```diff
@@ -0,0 +1,66 @@
+package com.example.uploadservice.s3
```

Comment:
I think s3 module is duplicated under two different paths 
android-upload-service/uploadservice-s3/ and android-upload-service/examples/app/uploadservice-s3. We only need the former.